### DISCUSSION TITLE
Improve zoom in iOS app?

### DISCUSSION THREAD

**Discussion by alstr** [tutao/tutanota#3410] [Ideas]
In the iOS app you have to double tap to zoom and (unless I’m missing something) you can’t zoom in any more than that.

Recently I needed to scan a barcode in an email but even when zoomed in it was too small. Admittedly this is bad email design, but having a standard pinch zoom with adjustable scale would be great. When I later opened the barcode in another app I was able to get it full screen and it scanned without problem.

**Comment by charlag** [comment]
Hi
It's tricky to do right now because we can't use native zoom but it would be good to have a better solution
